nuxeo-retention-web/search/retention/nuxeo-retention-search-results.html

```diff
@@ -22,15 +22,7 @@
 -->
 <dom-module id="nuxeo-retention-search-results">
   <template>
-    <style include="nuxeo-styles">
-      nuxeo-data-table,
```

Comment:
Don't we need this styling, at least for `nuxeo-data-table`?

Reply:
Yeah, this modification is not needed, it was copied past from my different tests. Thx